stable/prometheus-postgres-exporter/Chart.yaml

```
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: "0.8.0"
 description: A Helm chart for prometheus postgres-exporter
 name: prometheus-postgres-exporter
-version: 1.3.0
+version: 1.3.1
```

```suggestion
version: 1.4.0
```
this pr is adding some new feature for the chart not a bug fix

---

Done!